synapse/tests/test_neuron.py

```diff
@@ -255,6 +255,68 @@ def test_neuron_cell_notok(self):
                 self.raises(CellUserErr, user.open, addr, timeout=1)
                 self.raises(CellUserErr, user.open, addr, timeout=-1)
 
+    def test_neuron_double_initiate(self):
+        ''' Have the initiator send the listener two session initiation messages '''
+
+        with self.getTestDir() as dirn:
+
+            conf = {'bind': '127.0.0.1', 'host': 'localhost'}
+
+            with self.getLoggerStream('synapse.neuron') as stream, s_neuron.Cell(dirn, conf) as cell:
+
+                user = cell.celluser
+                addr = cell.getCellAddr()
+
+                with user.open(addr, timeout=2) as sess:
+                    sess._initiateSession()
+                    stream.wait(.1)
```

Comment:
stream.wait is only going to do anything if getLoggerStream() was called with a second parameter, otherwise it will just timeout and return None

Reply:
added second parameter. ty